tfplus/run_tfplus_precommit.sh

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 The TFPlus Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pip install pre-commit==2.21.0
```

Comment:
duplicate with dev/scripts/pre-commits.sh? 
merge them or rm the useless one.

Reply:
I have merged run_tfplus_precommit.sh into dev/scripts/pre-commits.sh.